swarm-production/docker-stack.yml

```diff
@@ -38,17 +39,21 @@ services:
           - node.labels.io.zenko.type == storage
 
   s3-front:
-    image: scality/s3server
+    image: zenko/cloudserver:pensieve-0
     ports:
-      - "8000"
+      - "8001"
```

Comment:
why changing from 8000?

Reply:
to make sure that incompatible versions won't work with this stack as this will be the number one problem. overall as long as it's coherent inside the stack the port does not matter